apps/emqx_bridge_kafka/src/emqx_bridge_kafka.erl

```diff
@@ -306,6 +307,10 @@ fields(kafka_producer_action) ->
         {tags, emqx_schema:tags_schema()},
         {description, emqx_schema:description_schema()}
     ] ++ producer_opts(action);
+fields(pre_configured_topic) ->
```

Comment:
there should be no need to pre-configure topics now?
given wolff now supports adding topics in the send2 call.